test/users.js

```diff
@@ -0,0 +1,200 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  should = chai.should(),
+  errors = require('../app/errors'),
+  User = require('../app/models').users;
+
+const validateError = ({ actualError, expectedMessage, expectedInternalCode }) => {
+  actualError.should.have.status(400);
+  actualError.response.should.be.json;
+  actualError.response.body.should.have.property('message');
+  actualError.response.body.should.have.property('internalCode');
+
+  const { message, internalCode } = actualError.response.body;
+  message.should.be.a('array');
+  message.should.have.lengthOf(1);
+  message[0].should.equal(expectedMessage);
+  internalCode.should.equal(expectedInternalCode);
+};
+
+describe('users', () => {
+  describe('/users POST', () => {
+    it('should fail because email is not wolox format', done => {
+      chai
+        .request(server)
+        .post('/users')
+        .send({
+          firstName: 'firstName',
```

Comment:
I noticed you are using this same parameters on several tests. Maybe it would be better to define a variable with these values at the beginning of the file, and just call it on each test. What do you think?